vonk/features/restful_api.rst

```diff
@@ -86,6 +86,34 @@ Vonk also supports ``_include:iterate`` and ``_revinclude:iterate``, as well as
       }
    },
 
+.. _restful_search_sort:
+
+Sorting
+^^^^^^^
+
+``_sort`` is implemented for searchparameters of types: 
+
+* string 
+* number 
+* uri
+* reference
+* datetime
+* token
+
+for the repositories:
+
+* SQL
+* SQLite
+* Memory.
+
+How is sort evaluated?
```

Comment:
We're in Europe so something should be mentioned about non-ASCII characters

Reply:
Good point. Depends on the collation you chose on the db server, but will add that.